### PR TITLE
#WIP move to Ingress

### PR DESCRIPTION
**Pull request by izakp**
Move from LoadBalancer service to ClusterIP internal service and add Ingress resource

Delete Nginx sidecar containers

### CODE REVIEW

File: default/hokusai/production.yml.j2 [hunk 1]
```diff
@@ -125,28 +94,31 @@ metadata:
...
+metadata:
+  name: {% raw %}{{ project_name }}{% endraw %}
+spec:
+  rules:
+    - host: {{ project_name }}.artsy.net
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: {% raw %}{{ project_name }}{% endraw %}-web
```
Comment (artsyjian) [inline default/hokusai/production.yml.j2:123]: [project]-web is the old loadBalancer service, correct? Should Ingress point to the new [project]-web-internal service instead?

Comment (izakp) [reply default/hokusai/production.yml.j2:123]: Yes - thanks good catch

Comment (izakp) [reply default/hokusai/production.yml.j2]: Resolved

**Comment by joeyAghion** [comment]
I didn't realize this was already in progress when I opened https://github.com/artsy/artsy-hokusai-templates/pull/26, but I think that supersedes this.

**Pull request closed without merging**